packages/web-app-files/tests/integration/specs/appBar.spec.js

```diff
@@ -37,12 +37,6 @@ describe('AppBar contains set of actions and informations', () => {
       render(
         AppBar,
         {
-          setup: () => {
```

Comment:
Not sure how to proceed here, we'd have to either render a full view or provide breadcrumb prop items to complete the rendering and fulfill the test criteria